apps/pie-docs/src/components/checkbox/motion/motion.md

```diff
@@ -0,0 +1,86 @@
+---
+eleventyNavigation:
+    key: Motion
+    parent: 'Checkbox'
+    order: 2
+shouldShowContents: true
+eleventyComputed:
+    hover: "{% include './hover.json' %}"
+    active: "{% include './active.json' %}"
+    selected: "{% include './selected.json' %}"
+    unselected: "{% include './unselected.json' %}"
+---
+
+## Example
+
+Illustrates the animations that happen in the checkbox component. In this component we have animations on hover and active states as well as when the component is selected and unselected.
```

Comment:
```suggestion
Illustrates the animations that happen in the checkbox component. In this component we have animations on hover and active states as well as when the component is selected and deselected.
```

Reply:
@xander-marjoram would this also be more appropriate for other usages of 'unselected'?

Reply:
Discussed on Slack. "Unselected" seems to be more regularly used in the tech context even if "deselected" has a more correct meaning in general.